simulation_parameters/Constants.cs

```diff
@@ -331,10 +331,9 @@ public static class Constants
     public const float MAX_OPACITY_MUTATION = 0.01f;
 
     // Mutation Variables
-    public const float MUTATION_BACTERIA_TO_EUKARYOTE = 1.0f;
-    public const float MUTATION_CREATION_RATE = 0.1f;
-    public const float MUTATION_EXTRA_CREATION_RATE = 0.1f;
-    public const float MUTATION_DELETION_RATE = 0.1f;
+    public const float MUTATION_BACTERIA_TO_EUKARYOTE = 0.01f;
```

Comment:
I didn't change the chance of evolving a nucleus; I just made it a portion like everything else instead of a percentage.